feature-libs/cart/main/root/components/mini-cart/mini-cart.module.ts

```diff
@@ -17,6 +18,7 @@ import { MiniCartComponent } from './mini-cart.component';
       cmsComponents: {
         MiniCartComponent: {
           component: MiniCartComponent,
+          providers: [{ provide: MiniCartComponentService }],
```

Comment:
`MiniCartComponentService` is `providedIn: 'root'`, so we don't need provide it in component, right?

Reply:
fixed